app/src/main/res/values/strings.xml

```diff
@@ -138,6 +138,7 @@
     <string name="dialogs_top_year">Top Year</string>
     <string name="dialogs_old">Old</string>
     <string name="dialogs_unread">Unread</string>
+    <string name="dialogs_server_version_outdated">Server version (%1$s) is under the minimum (%2$s).\n\nPlease inform your administrator and login to another instance, or sign out and use the default instance.</string>
```

Comment:
I would make a small change here:

"is under the *supported* minimum"

Reply:
So, apparently supporting older versions isn't that painful. So maybe we should fix the wording even more to let users know that most features should work, but that they should ask the admin of their instance to upgrade.

Reply:
Wonderful, in that case it might be better to separate the supported versions (0.18), temporary supported versions (0.17.X) and not supported at all (0.16 and below) That would cover all  #673

Reply:
I think it increases testing load too much per release. I don't think anyone is particularly interested in testing on older server versions, so merely stating that it's unsupported and some features may not work should be sufficient.

That said, if someone builds a mock "server" or something, we could at least write some unit tests. I might get around to that eventually since I'm interested in testing, but there's a lot of other missing functionality I'd also like to work on.

If we decide to officially "partially" support the previous release, we'll need to store the dismissal in the database. I intentionally didn't do that here to be just that much more annoying to get users to pressure their admins to upgrade.

I'm happy either way, I just don't want to set expectations we can't really meet.

Reply:
I didn't mean to imply we should fully support 0.17.X or any future outdated version, Just that 16.x is not supported at all  thus should not be reported at all in the issues.

Reply:
Ok, fair. I don't know what changed between 0.16 to 0.17, so I guess we could handle that as a special case.

I'm more interested in what we do going forward.